frontend/libs/studio-components/src/components/StudioRecommendedNextAction/StudioRecommendedNextAction.tsx

```diff
@@ -0,0 +1,46 @@
+import { StudioButton } from '../StudioButton';
+import React from 'react';
+import classes from './StudioRecommendedNextAction.module.css';
+import { StudioCard } from '../StudioCard';
+
+export interface StudioRecommendedNextActionProps {
```

Comment:
I think we mostly are using `type` instead of `interface`. 😊 
```suggestion
export type StudioRecommendedNextActionProps {
```